R/latent_individual.R

```diff
@@ -151,6 +151,11 @@ epidist_formula.epidist_latent_individual <- function(data, delay_central = ~ 1,
 #' @family latent_individual
 #' @export
 epidist_family.epidist_latent_individual <- function(data, family = "lognormal",
+                                                     dpars = c("mu", "sigma"),
```

Comment:
I think mu and sigma are the standard names brms always uses so we might want to keep them?

Reply:
This is the crux of the issue. What are `mu` and `sigma` for e.g. the Gamma distribution? My understanding is that `mu` is always there,  but I don't know that `sigma` is. For the Gamma I think it's `mu` and `shape` ($\alpha > 0$). More broadly I think it's `mu` plus other things.

Reply:
no sorry. As in we keep the names and define them differently. Or we define them as brms does and then need to think about interface in more detail. The thing we don't want to do is try and transform in some mad way!

Reply:
> As in we keep the names and define them differently

I think this leads to solutions like "The thing we don't want to do is try and transform in some mad way!".

> Or we define them as brms does and then need to think about interface in more detail

I feel like this is the direction needed.